src/stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/types/search.py

```diff
@@ -129,13 +148,29 @@ def filter_fields(self) -> Dict:
         }
 
 
-class SQLAlchemySTACSearch(Search):
+class SQLAlchemySTACSearch(BaseModel):
     """Search model."""
 
+    ids: Optional[List[str]]
+    bbox: Optional[BBox]
+    intersects: Optional[
+        Union[
+            Point,
+            MultiPoint,
+            LineString,
+            MultiLineString,
+            Polygon,
+            MultiPolygon,
+            GeometryCollection,
+        ]
+    ]
+    datetime: Optional[str]
+    sortby: Optional[List[SortExtension]]
+
     # Make collections optional, default to searching all collections if none are provided
     collections: Optional[List[str]] = None
     # Override default field extension to include default fields and pydantic includes/excludes factory
```

Comment:
Please delete comments that are no longer relevant

Reply:
Managed to mark the wrong lines. I meant for instance "override default..." which doesnt make sense, when we are not inheriting anything which can be overridden.